server.js

```diff
@@ -7062,59 +7062,64 @@ cache(function(data, match, sendBadge, request) {
 // Libraries.io integration.
 camp.route(/^\/librariesio\/(github|release)\/([\w\-_]+\/[\w\-_]+)\/?([\w\-_.]+)?\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
+  const resource  = match[1];
+  const project   = match[2];
+  const version   = match[3];
+  const format    = match[4];
 
-  var resource  = match[1];
-  var project   = match[2];
-  var version   = match[3];
-  var format    = match[4];
-
-  var uri;
+  let uri;
   switch (resource) {
-    case 'github':
+    case 'github': {
       uri = 'https://libraries.io/api/github/' + project + '/dependencies';
       break;
-    case 'release':
-      var v = version || 'latest';
+    }
+    case 'release': {
+      const v = version || 'latest';
       uri = 'https://libraries.io/api/' + project + '/' + v + '/dependencies';
       break;
+    }
   }
 
-  var options = {method: 'GET', json: true, uri: uri};
-  var badgeData = getBadgeData('dependencies', data);
+  const options = {method: 'GET', json: true, uri: uri};
+  const badgeData = getBadgeData('dependencies', data);
 
   request(options, function(err, res, json) {
-
     if (err || res.statusCode !== 200) {
```

Comment:
Would you like to improve error handling by using 
https://github.com/badges/shields/blob/e1755df212ff63b8ec43167e67000d1bbecf429d/lib/error-helper.js#L8
in this PR?

Reply:
Done, thanks for reviewing!